client/src/components/TaskManager/TaskManger.jsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+import React, { useState, useRef } from "react";
+import "./taskmanager.css";
+import { IoIosAddCircleOutline } from "react-icons/io";
+import { RiDeleteBin5Line } from "react-icons/ri";
+
+function TaskManager() {
+  const [currentTab, setCurrentTab] = useState("todo");
+  const [newTaskText, setNewTaskText] = useState("");
+  const [isInputVisible, setInputVisible] = useState(false);
+  const [buttonVisible, setButtonVisible] = useState(true);
+  const [tasks, setTasks] = useState({
```

Comment:
Fetch the tasks from the api inside a useEffect to update your state

Reply:
done